codespell_lib/data/dictionary.txt

```diff
@@ -18459,13 +18459,13 @@ maxinum->maximum
 maxium->maximum
 maxiumum->maximum
 maxmimum->maximum
-maxosx->macosx
+maxosx->macOS X
```

Comment:
We need to offer the correction without spaces first, or you'll break a lot of code related stuff:
```suggestion
maxosx->maxosx, macOS X,
```

You could probably put this on it's own in the code dictionary:
`maxosx->macOS X`